app/src/userpages/tests/components/KeyField/KeyField.test.jsx

```diff
@@ -63,27 +64,6 @@ describe('KeyField', () => {
 
             const action = shallow(el.find(TextInput).prop('actions')[0])
             assert(action.find('Translate').shallow().text() === 'reveal')
-
-            el.instance().toggleHidden()
```

Comment:
Tests removed because `el.instance()` doesn't work and using `action.simulate('click')` results in an error inside `reactstrap`'s `DropdownItem` component (see https://github.com/reactstrap/reactstrap/issues/1457).

Reply:
blegh